Misc/NEWS.d/next/Tests/2025-06-11-16-52-49.gh-issue-135401.ccMXmL.rst

```diff
@@ -0,0 +1,3 @@
+Add a new GitHub CI job to test the :py:mod:`ssl` module with AWS-LC__ as the backing cryptography and TLS library.
+
+__ https://github.com/aws/aws-lc
```

Comment:
Use an inline link instead of an anonymous one. This can be achieved as follows IIRC (check if this is correct) :

```rst
`text <link>`_
```

Reply:
yep, that [looks right](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases).